docs/SETUP.md

```diff
@@ -176,27 +96,19 @@ In your `/etc/nginx/sites-enabled/default`, remove the `location /` block, and a
 
 In order to make sure you didn't mess anything up, run:
 
-```sh
-$ sudo nginx -t
```

Comment:
I think it's helpful to run this, because it will give a clear error if someone made a typo or something